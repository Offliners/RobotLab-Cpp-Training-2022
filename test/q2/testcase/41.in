232
MUrDER
sHinY
MovE
CONSERVATION
IMAge
ACHE
PONY
EVOLVe
dynamic
vaLUablE
GENERALIzE
NAivE
CoMPARATive
sOciology
DweLler
MUSIC
Picture
PAnORaMa
CentiGrAde
CoUrt
ShEEp
DiSCOVER
pleAsant
KNIGHT
attempt
LESSER
TRADITIoNAL
DIVERSE
DeTAILED
OUtStaNding
constRUCtiON
hiNT
SomEtiMes
SNOWY
PhYsIcAl
aging
GAUGE
MONEtARy
3E190870
SUPERINTEnDeNT
DISSATISfY
reFun4
LAmp
WHerE
coTTAGE
bUreau
INTELLECT
paper
ABLE
B463AUC7Ac3
PLedgE
cooperate
BUll
TRUNK
GOODBYE
tEndeR
fiu
composite
HAWk
continue
dyE
CLOSE
ARChItect
bUg
IntErMediAtE
SECRET
BEG
grocery
SupeRvISE
PERCent
cHANCellOR
PaVEMEnT
3o538
faShiON
sOvereigN
mASTER
mystErioUs
BIBLical
NOVEmBeR
HOWL
famIliar
ORNAMENT
demon
middLE
AnAloGy
stAtIon
disgust
SERUM
tablespoon
SysTEMaTiC
PoST
MINorITY
CONVeNe
oVERheAr
glUE
nickname
ve800l6
LawnMOwer
oThERwiSe
eLeCTricIty
HUMBLE
statESman
homoSexual
cApTioN
TIMbEr
cleaniNg
altituDe
fOrTh
SpECTrA
HOW
moan
dIsHoNEst
liTtEr
bONuS
ChristIanity
dISTURb
KNOW-HOW
FOREST
tErRiBlE
cEase
promIsE
DiagrAm
jUSTICE
LanDiNG
AMBitIOus
PLUS
cOMPLeXIoN
ReFER
matter
painfully
REhabIliTaTiON
induStrY
SaNction
hunter
MeNu
ACCUMULATiON
awarD
sWeAT
mORaL
ENJOYABLE
sOvereignty
RESORT
solId
SIngULAr
sUdDEn
HeaDqUARTeRS
irRItaTe
939t917
FOUR
cURTAin
aMMunITiON
maLL
swEET
HOMELAND
INAPpROPRIATE
forWaRd
POSTeR
darkroom
UNEMPLOYED
RefuSal
DREAD
OUTBREAK
WIdE
insecURe
hail
Bland
COLleCTOr
CLOCK
stREsS
magnifY
aparTment
hoNEyMOON
LikinG
stone
aGAINST
PRoGRAM
HOLY
HeaVEN
cOnsole
InteRPRet
sport
aPpEaraNcE
BEAr
SOUTHErN
LOGiC
clockwise
ALik5
c
sTRivE
altERation
exhIbitIOn
tAxI
literacy
OvERCOAT
VOlUme
BReeDeR
psYchIaTriSt
malaria
COntribUtE
CLiMATE
bIoCHeMistry
DIAMoND
RevIsion
grasshopper
ATTACK
bELOW
LAST
PREseRVE
DISPLEASE
reCent
combine
cOPE
sTOol
muddY
NOD
RAGE
golDfish
PREOCCUPY
BREAth
EXPLOSIVE
MiNistrY
gENERATOR
4B2186o4
eneRgy
pancake
FounTAIN
capE
equaliTy
dictionary
inner
lOVE
rEactionAry
