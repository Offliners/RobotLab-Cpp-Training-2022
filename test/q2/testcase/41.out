n./v. 謀殺
adj. 閃爍的
v.(n. )使移動；使感動
n. 保護
n. 影像；形像
n./v. 疼痛
n. 小馬
v. 使逐漸發展
adj. 動力的；動態的
adj. 珍貴的
v. 歸納
adj.  天真的；樸素的
adj. 相比而言的
n. 社會學
n. 居民
n. 音樂
n./v. 圖畫；想像
n. 全景
adj./n. 攝氏(的)
n. 法庭
n. 羊
v. 發現
adj. 令人愉快的
n. 騎士；爵士
v./n. 嘗試
adj./adv. 較小的
adj. 傳統的
adj. 各種各樣的
adj. 詳細的
adj. 傑出的
n. 建造物；建設
n./v. 暗示
adv. 有時
adj. 多雪的
adj. 身體的
adj. 變老的
n. 標準度量
adj. 貨幣的
Unknown
n. 監護人；警長
v. 使不快
Unknown
n. 燈
adv./pron./conj.  在哪裡；哪裡
n. 農舍；小屋
n. 局；處；署
n. 理解力
n. 紙
adj. 能；有能力的
Unknown
v. 發誓；保證
v. 合作
n. (未閹割的) 公牛
n. 樹幹
interj./n. 再見
adj. 嫩的
n. 流行性感冒
adj. 混合成的
n. 鷹
v. 持續
v./n. 染料；染色
adj./adv. 接近的
n. 建築師；設計師
n. 蟲子
adj. 中問的
adj./n. 祕密的
v. 乞求；乞討
n. 食品雜貨店
v. 監督
n. 百分之…
n. 司法官
n. 人行道
Unknown
n./v. 時尚；風氣
n. 元首
n. 主人；雇主
adj. 神祕的
adj. 聖經的
n. ＋一月
v. 號叫
adj. 熟悉的
n. 裝飾
n. 惡鬼
adj./n. 中問的；中部
n. 相似
n. 車站
n./v. 嫌惡；使厭惡
n. 血清
n. 大湯匙
adj. 有系統的
n. 位置；郵局
n. 少數
v. 召集
v. 偷聽；無意中聽到
n./v. 膠水；黏牢
n. 綽號
Unknown
n. 除草機
adv. 除此之外
n. 電；電流
adj. 謙恭的
n. 政治家
adj./n. 同性戀
n. 說明文字
n. 木材
n. 掃除
n. 高度
adv. 向前
n. 光譜
adv. 如何；多少
v./n. 呻吟(聲)
adj. 不誠實的
n. 雜物；垃圾
n. 紅利
n. 基督教
v. 妨礙；打擾
n. 技能
n. 森林
adj. 可怕的
v. 停止
n./v. 諾言；許諾
n. 圖表(解)
n. 公平；正義
n. 登陸
adj. 野心勃勃的
prep./n./adj. 加
n. 氣色
v. 起因於…
n. 事情；問題
ad v. 痛苦地
n. 康復
n. 工業
n. 批准
n. 獵人
n. 菜單；項目單
n. 累積
v. /n授予；獎品
n./v. 汗水；出汗
adj. 道德上的
adj. 有樂趣的
n. 主權
v. 前往
adj. 堅固的
adj. 單數的
adj. 突然的
n. 總部
v. 激怒
Unknown
adj./n. 四
n./v. 簾；給…裝上簾子
n. 彈藥
n. 大規模購物中心
adj. 甜的
n. 祖國；家夠卜
adj. 不適當的
adj./v. 向前；今後
n. 海報
n. 暗房
adj. 失業的
n. 拒絕
n./v. 害怕
v. 爆發；發生
adj. 寬的
adj. 不安全的
v. 歡迎
adj. 溫和的
n. 收藏者
n. 時鐘
n./v. 壓力
v. 擴大；誇大；誇獎
n. 公寓大樓
n. 蜜月假期
n. 愛好
n./v. 石頭
prep. 反對；對著
n./v. 程序；編製程序
adj. 神聖的
n. 天國；極樂之地
v. 安慰
v. 解釋；翻譯
n. 運動
n. 外貌；顯露
n. 熊；魯莽之人
adj. 南方的
n. 邏輯
adj. 順時針方向的
Unknown
Unknown
v. 奮鬥
adj. 更動
n. 展示會
n. 計程車
n. 識字；讀寫能力
n. 外套大衣
n. 音量
n. 繁殖者
n. 精神科醫生
n. 瘧疾
v. 捐獻；貢獻
n. 氣候；風氣
n. 生物化學
n. 鑽石
n. 修正
n. 蝗蟲；蚱蜢
v./n. 攻擊
prep./adv. 在…下方
adj./adv./n./pron.  最後的；最近的
v. 保存
v. 使不快
adj. 最近的
v. 結合
v. 競爭；妥善處理
n. 凳子
adj. 泥的；骯髒的
n./v. 點頭；打瞌睡
n./v. 發怒
n. 金魚
v. 使全神貫注
n. 呼吸
adj./n. 爆炸性的
n. 牧師之職；內閣
n. 發電機
Unknown
n. 活力
n. 煎餅
n. 噴泉；噴水池
n. 披肩；海角
n. 相等；平等；均等
n. 字典
adj. 內在的
v./n. 愛；喜歡
adj. 保守的；反動的
